projects/services-ogc/package.json

```diff
@@ -20,7 +20,7 @@
     "ol": "0.0.0-PLACEHOLDER-VENDOR",
     "@michaellangbein/jsonix": "0.0.0-PLACEHOLDER-VENDOR",
     "tslib": "0.0.0-PLACEHOLDER-VENDOR",
-    "opensearch-browser": "0.0.0-PLACEHOLDER-VENDOR"
+    "opensearch-browser": "2.0.0-alpha.5"
```

Comment:
This should be `0.0.0-PLACEHOLDER-VENDOR` because it gets replaced before publish and is already installed in the main package with `2.0.0-alpha.5`

Reply:
Sorry, that was by accident - a merge conflict improperly resolved. Has been adjusted.